docs/native-modules-android.md

```diff
@@ -119,236 +132,420 @@ import java.util.ArrayList;
+    List<ReactPackage> packages = new PackageList(this).getPackages();
+    // below MyAppPackage is added to the list of packages returned
+    packages.add(new MyAppPackage());
+    return packages;
+  }
+```
 
-...
-import com.your-app-name.CustomToastPackage; // <-- Add this line with your package name.
-...
+You have now successfully registered your native module for Android!
 
-protected List<ReactPackage> getPackages() {
-  @SuppressWarnings("UnnecessaryLocalVariable")
-  List<ReactPackage> packages = new PackageList(this).getPackages();
-  // Packages that cannot be autolinked yet can be added manually here, for example:
-  // packages.add(new MyReactNativePackage());
-  packages.add(new CustomToastPackage()); // <-- Add this line with your package name.
-  return packages;
-}
+### Test What You Have Built
+
+At this point, you have set up the basic scaffolding for your native module in Android. Test that out by accessing the native module and invoking its exported method in JavaScript.
+
+Find a place in your application where you would like to add a call to the native module’s `createCalendarEvent` method. Below is an example of a component, `NewModuleButton` you can add in your app. You can invoke the native module inside `NewModuleButton`'s `onPress` function.
+
+```jsx
+import React from 'react';
+import { NativeModules, Button } from 'react-native';
+
+const NewModuleButton = () => {
+  const onPress = () => {
+    console.log('We will invoke the native module here!');
+  };
+
+  return (
+    <Button
+      title="Click to invoke your native module!"
+      color="#841584"
+      onPress={onPress}
+    />
+  );
+};
+
+export default NewModuleButton;
+```
+
+In order to access your native module from Javascript you need to first import `NativeModules` from React Native:
+
+```jsx
+import { NativeModules } from 'react-native';
+```
+
+You can then access the `CalendarModule` native module off of `NativeModules`.
+
+```jsx
+const { CalendarModule } = NativeModules;
+```
+
+Now that you have the CalendarModule native module available, you can invoke your native method `createCalendarEvent`. Below it is added to the `onPress` method in `NewModuleButton`:
+
+```jsx
+const onPress = () => {
+  CalendarModule.createCalendarEvent('testName', 'testLocation');
+};
 ```
 
-To access your new functionality from JavaScript, it is common to wrap the native module in a JavaScript module. This is not necessary but saves the consumers of your library the need to pull it off of `NativeModules` each time. This JavaScript file also becomes a good location for you to add any JavaScript side functionality.
+The final step is to rebuild the React Native app so that you can have the latest native code (with your new native module!) available. In your command line, where the react native application is located, run the following:
 
-Create a new JavaScript file named `ToastExample.js` with the content below:
+```jsx
+react-native run-android
```

Comment:
Same like in iOS guide, please use the `npx` for the run command.

Reply:
done